portfolio/src/test/java/com/google/sps/LinkedListTimeRangeGroupTest.java

```diff
@@ -0,0 +1,557 @@
+package com.google.sps.data;
```

Comment:
These are exactly the same tests as ArrayListTimeRangeGroupTest.java which is already in master except this tests LinkedListTimeRangeGroup instead.